plugins/polio/js/src/pages/Dashboard.js

```diff
@@ -190,7 +190,7 @@ const Dashboard = ({ router }) => {
                     const allRounds = (
                         <>
                             {row.rounds.map(r => (
-                                <li>
+                                <li key={`${r.number}-${r.started_at}`}>
```

Comment:
This is also fixed on my PR [here](https://github.com/BLSQ/iaso/pull/17)

Reply:
next week ;-)